etc/ci/performance/firefox/revision_46.0.1.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
`package.json` is the minimal required version. Here we put a the 46.0.1 json because we are currently comparing Servo with Gecko 46.0.1. I guess we can remove this because I don't want to support Gecko testing in servo/servo right now.